frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -50,25 +47,24 @@ const getPluginTabPages = (tabs: DashboardsTab[], cards: DashboardsCard[]): Page
 const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight, k8sModels }) => {
   const { t } = useTranslation();
   const title = t('dashboard~Overview');
+
+  /* To add a new tab next to cluster overview */
```

Comment:
This cannot be commented out. We want to keep the ability to add new tabs to cluster overview

Reply:
ack... I will revert the changes...